user.go

```diff
@@ -74,8 +75,14 @@ type User struct {
 
 	// ProfileURI is a pointer to where their profile picture lives
 	ProfileURI string `bson:"profileUri" json:"profileUri,omitempty" xml:"profileUri,omitempty"`
+
+	// mfaFactors contains the MFA factors the user is signed up to
+	MfaFactors map[string]MultiFactorType `bson:"MultiFactorType" json:"MultiFactorType,omitempty" xml:"MultiFactorType,omitempty"`
```

Comment:
lang: to keep consistent with camelCase, we should lowercase the first letter in these struct tags. As for the attribute naming in Go, where we have a known acronym we tend to leave it fully uppercase, like HTTP so in this case we can use MFA.

```suggestion
	// MFAFactors contains the MFA Factor ID to the type of factor the user is signed up to.
	MFAFactors map[string]MultiFactorType `bson:"multiFactorType" json:"multiFactorType,omitempty" xml:"multiFactorType,omitempty"`
```